code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -27,6 +27,7 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	var/volume = CELL_VOLUME //liters
 	var/last_share = 0
 	var/list/reaction_results
+	var/static/list/nonreactive_gases = list(typecacheof(/datum/gas/oxygen, /datum/gas/nitrogen, /datum/gas/carbon_dioxide)) // These gasses cannot react amongst themselves

```

Comment:
Why not
```
GLOBAL_LIST_INIT(nonreactive_gases, list(typecacheof(/datum/gas/oxygen, /datum/gas/nitrogen, /datum/gas/carbon_dioxide)))
```
?

Reply:
Seems rather odd to have such a list here in the first place, instead of checking already defined gasses for a nonreact flag.

Reply:
I think it maybe due to what is technically faster per proc call